src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -514,7 +522,7 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
   }
 
   /** Should fail if a value is already set.  The keys in the Map are locally qualified names. */
-  override def setOutputs(workflowId: WorkflowId, key: OutputKey, callOutputs: WorkflowOutputs): Future[Unit] = {
+  override def setOutputs(workflowId: WorkflowId, key: OutputKey, callOutputs: WorkflowOutputs, topLevelOutputs: Map[FullyQualifiedName, WdlType]): Future[Unit] = {
```

Comment:
I'm wondering if it wouldn't be cleaner to have `WorkflowOutputs` be a real `case class` instead of a `type` and have a boolean "I am a workflow output: yes or no". Or even the Jeff way with sealed trait and 2 case classes ?